.github/workflows/rust.yml

```diff
@@ -1,4 +1,8 @@
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
```

Comment:
TBH, I personally prefer to have all pushes tested, even if I’m not ready yet to create a PR for that. Also, it does provide real value IMO to see if a commit on its own is failing, or the *merge with master* is. The redundant work is unfortunate though.

Reply:
Draft PRs do also get tested with this config i think.  Is that not sufficient?

I'm not a huge fan of having all checks run twice and double reporting errors isn't very user friendly.

Reply:
hm, I think I can agree with you. Rather to just open a draft PR than to have "hidden" branches only for testing stuff. I think I can change this for native as well, or also just migrate over to github actions while at it.